source/services/intermediateCredentials.ts

```diff
@@ -63,6 +67,15 @@ export async function storeCredentialsForVault(source: VaultSource, password: st
             }
         };
     }, {});
+
+    // Prompt the user to set Buttercup as AutoFill provider in system settings
+    // This is likely NOT the correct place to trigger this - it should be in response to some UI
+    // that explicitly advises what autofill is and why they should enable it etc.
```

Comment:
@perry-mitchell Please note this comment. I've placed the autofill settings prompt here just to prove it's working, but it's probably not very good UX to have the prompt take over the screen as the user is unlocking their vault with no fore-warning

Reply:
Yeah you're right, I did hastily add autofill so that it runs all of the time, without asking the user to allow it. I should move it to a switch in the settings.